modules/distr-tracing-features.adoc

```diff
@@ -1,7 +1,7 @@
 ////
 This module included in the following assemblies:
 -service_mesh/v2x/ossm-architecture.adoc
--dist_tracing_arch/distr-tracing-architecture.adoc
+-observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc
```

Comment:
Globally, the module file metadata needs a *, not a - in the list: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
@skrthomas, yes, thank you, I am aware of this, and most of the files in this PR already comply with this rule. However, there has been a number of old files that I had to move recently, and these were initially drafted by a developer, hence the different metadata format. These files are going away in a few months because the component they document has already been deprecated and is waiting for removal. In any case, I updated the metadata lines in the following files as you pointed out:
```
        modified:   modules/distr-tracing-accessing-jaeger-console.adoc
        modified:   modules/distr-tracing-architecture.adoc
        modified:   modules/distr-tracing-config-default.adoc
        modified:   modules/distr-tracing-config-ingester.adoc
        modified:   modules/distr-tracing-config-jaeger-collector.adoc
        modified:   modules/distr-tracing-config-query.adoc
        modified:   modules/distr-tracing-config-sampling.adoc
        modified:   modules/distr-tracing-config-storage.adoc
        modified:   modules/distr-tracing-deploy-default.adoc
        modified:   modules/distr-tracing-deploy-production-es.adoc
        modified:   modules/distr-tracing-deploy-streaming.adoc
        modified:   modules/distr-tracing-deployment-best-practices.adoc
        modified:   modules/distr-tracing-features.adoc
        modified:   modules/distr-tracing-install-elasticsearch.adoc
        modified:   modules/distr-tracing-install-jaeger-operator.adoc
        modified:   modules/distr-tracing-install-overview.adoc
        modified:   modules/distr-tracing-removing-instance-cli.adoc
        modified:   modules/distr-tracing-removing-instance.adoc
        modified:   modules/distr-tracing-sidecar-automatic.adoc
        modified:   modules/distr-tracing-sidecar-manual.adoc
        modified:   modules/distr-tracing-upgrading-es5-es6.adoc
```

Reply:
Thanks for updating these and the clarifications. Apologies to ask you to update files that are going to be removed soon. I can appreciate why you didn't update them in this case. Sorry...its hard to know these things as a peer reviewer :) For future, if you have things like this that are known diversions for a reason, its helpful to leave a review comment in the PR at the place where the diversion occurs, that way we don't burden you with unnecessary feedback.